suggestion/README.md

```diff
@@ -46,11 +49,19 @@ _Note:_ Among the **above** variables all channels can be separate or same. All
 
 7. `$Upvote` -
 
-   The emoji for upvote reaction. Supports both custom as well as inbuilt emojis. For inbuilt emojis, it is recommended to use corresponding unicode characters. Example : "👍".
+   The emoji for upvote reaction. Supports both custom as well as inbuilt emojis. For inbuilt emojis, it is recommended to use corresponding unicode characters such as `👍`.
 
 8. `$Downvote` -
 
-   The emoji for downvote reaction. Supports both custom as well as inbuilt emojis. For inbuilt emojis, it is recommended to use corresponding unicode characters. Example : "👎".
+   The emoji for downvote reaction. Supports both custom as well as inbuilt emojis. For inbuilt emojis, it is recommended to use corresponding unicode characters, like `👎`.
```

Comment:
```suggestion
   Downvote reaction emoji to use; inbuilt and custom emojis are both supported. If using a custom emoji, use the `name:id` / `a:name:id` format (for animated emojis). Otherwise, use the corresponding Unicode character, for example, `👍`.
```